app/(afterLogin)/(compensation)/deduction/[id]/page.tsx

```
@@ -24,9 +24,13 @@ const SingleDeductionPage = () => {
       setIsAllowanceGlobal(false);
     }
   }, [deductionData, setIsAllowanceGlobal]);
+  const { employeeBenefitData } = useBenefitEntitlementStore();
+
 
   return (
     <div className="bg-white rounded-lg px-1 py-4 sm:p-6">
+  <div>
```

ESLint

[error] Insert `····`

(prettier/prettier)